event_seat_reserve/models/event_ticket.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
+from odoo import api, fields, models
+
+
+class EventTicket(models.Model):
+    _inherit = "event.event.ticket"
+
+    seats_reserved_unconfirmed = fields.Integer(
+        string="Number of Reserved Attendees", compute="_compute_seats"
+    )
+
+    @api.depends("seats_max", "registration_ids.state", "registration_ids.active")
+    def _compute_seats(self):
+        """Determine reserved, available, reserved but unconfirmed and used seats."""
+        # initialize fields to 0 + compute seats availability
+        res = super()._compute_seats()
+        counts = dict()
```

Comment:
This compute method is not covered by tests.
Could we add a test to check the `seats_reserved_unconfirmed` value?